docker-compose.yaml

```diff
@@ -143,13 +143,31 @@ services:
         condition: service_healthy
 
   front-end:
-    build: ./frontend
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile.dev
```

Comment:
I don't think this was intended, not sure why the switch to dev container on the prod Compose